### PR TITLE
EES-4280 - fix UI tests

### PR DESCRIPTION
**Pull request by luke-h1**
This PR: 
fixes the majority of the UI test failures seen over the past 4 weeks
ui-test-report

### CODE REVIEW

File: tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot [hunk 1]
```diff
@@ -315,6 +321,8 @@ Check that superseded warning link takes user to superseding-publication data-ca
     user checks summary list contains    Publication    ${PUBLICATION_NAME_ARCHIVE}
 
     user waits until page contains    Choose a release
+
+    # failing here
```
Comment (duncan-at-hiveit) [inline tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot:325]: Has this comment been left in on purpose?

File: tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot [hunk 2]
```diff
@@ -302,11 +302,17 @@ Check data catalogue page contains superseded warning for archived publication (
     user checks page contains element    testid:superseded-by-link
 
 Check that superseded warning link takes user to superseding-publication data-catalogue step 1
+    [Documentation]    Failing due to https://dfedigital.atlassian.net/browse/EES-4269
```
Comment (duncan-at-hiveit) [inline tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot:305]: Is it possible to add a warning to the descriptions of these bug tickets to look for Robot Tests that mention their ticket ref, and to include them again as a part of the bug fixing please?

**Review by duncan-at-hiveit — APPROVED** [review]

File: tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot (hunk 2 above)
Comment (N-moh) [reply tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot:305]: Good Idea. Done.

Comment (luke-h1) [reply tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot:305]: That's what the [Documentation] tag is for. For these types of issues, we typically comment on the JIra  ticket to remove the Failing tag when the bug has been fixed

File: tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot (hunk 1 above)
Comment (luke-h1) [reply tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot:325]: Yes, just to denote that the test specifically fails on this line

**Pull request merged**